.github/workflows/AddPRComment.yml

```diff
@@ -8,6 +8,8 @@ permissions:
 jobs:
   thank-user:
     runs-on: ubuntu-20.04
+    permissions:
+      pull-requests: write
```

Comment:
```suggestion
```
This workflow has already been configured with the `pull-requests: write` permission at the workflow level in lines 5-6. There's no need to re-add this permission at the job level. You can align with `AddIssueComment.yml` to add the permission to either the workflow or job level.